spec/unit/hanami/cli/commands/app/generate/operation_spec.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+
+require "hanami"
+
+RSpec.describe Hanami::CLI::Commands::App::Generate::Operation, :app do
+  subject { described_class.new(fs: fs, inflector: inflector, generator: generator) }
+
+  let(:out) { StringIO.new }
+  let(:fs) { Hanami::CLI::Files.new(memory: true, out: out) }
+  let(:inflector) { Dry::Inflector.new }
+  let(:generator) { Hanami::CLI::Generators::App::Operation.new(fs: fs, inflector: inflector) }
+  let(:app) { Hanami.app.namespace }
+  let(:dir) { inflector.underscore(app) }
+
+  def output
+    out.rewind && out.read.chomp
+  end
+
+  context "generating for app" do
+    it "generates an operation" do
+      subject.call(name: "add_book")
+
+      operation_file = <<~EXPECTED
+        # frozen_string_literal: true
+
+        module Test
+          module Operations
+            class AddBook < Test::Operation
+              def call(input)
+              end
```

Comment:
I worry about `call(input)` as a default signature making it feel like a single positional param is the only way this `#call` method can work (which is certainly not the case). What do you think about making it just `def call`?